packages/wrapped-keys-lit-actions/src/index.ts

```diff
@@ -1,12 +1,16 @@
+import * as batchGenerateEncryptedKeys from './generated/common/batchGenerateEncryptedKeys';
 import * as exportPrivateKey from './generated/common/exportPrivateKey';
 import * as generateEncryptedEthereumPrivateKey from './generated/ethereum/generateEncryptedEthereumPrivateKey';
-import * as signMessageWithEthereumEncryptedKey from './generated/ethereum/signMessageWithEthereumEncryptedKey';
-import * as signTransactionWithEthereumEncryptedKey from './generated/ethereum/signTransactionWithEthereumEncryptedKey';
+import * as signMessageWithEthereumEncryptedKey from './generated/ethereum/signMessageWithEncryptedEthereumKey';
```

Comment:
Renamed the files for least-to-most-specific naming convention. Exports remain named the same until we ship a breaking change, then we'll make those consistent with the filenames.